docs/deployment.md

```diff
@@ -250,6 +250,30 @@ Clients must connect through the HTTPS port using the certificates with which th
 was configured. Currently, the certificates used by the Helm chart can be found in the
 `deployments/helm/onos-config/files/certs` directory.
 
+
+### Command-line shell container
+
+For containerized environments like Kubernetes, a Docker image `onosproject/onos-cli` is provided.
+This image is built as part of the normal build.
+
+To use the CLI in Kubernetes, run the `onosproject/onos-cli` image in a single pod deployment:
+
+```bash
+> kubectl run onos-cli --rm -it --image onosproject/onos-cli:latest --image-pull-policy "IfNotPresent" --restart "Never"
+```
+This command will run the CLI image as a Deployment and log into the bash shell.
+Once you've joined the container, you can connect to the `onos-config` server by running:
+
+```bash
+> onos config set address something-else-onos-config:5150
+something-else-onos-config:5150
```

Comment:
why is it needed two times ? also can you provide a working command assuming onos-config as the name of the deployment ?

Reply:
The command outputs that. Only the line prefixed with > is the CLI (unless it would have \ at the end)